pkgs/by-name/pc/pcsx2/package.nix

```diff
@@ -36,13 +37,25 @@ let
     qtwayland
     wrapQtAppsHook
     ;
+
+  cubeb' = cubeb.overrideAttrs (old: {
+    patches = (old.patches or [ ]) ++ [
+      (fetchpatch {
+        url = "https://github.com/PCSX2/pcsx2/commit/430e31abe4a9e09567cb542f1416b011bb9b6ef9.patch";
+        stripLen = 2;
+        hash = "sha256-bbH0c1X3lMeX6hfNKObhcq5xraFpicFV3mODQGYudvQ=";
+      })
+    ];
```

Comment:
this strikes me as brittle, would you be willing to add `pcsx2` to `cubeb.passthru.tests` with a comment of it being due to supporting this patch?